py/client/tests/test_session.py

```diff
@@ -221,7 +221,7 @@ def test_input_table(self):
         dh_table = self.session.import_table(pa_table)
```

Comment:
Should we unit test any of the other cases that were changed?  I don't have a strong view, and you could easily argue that "no" is the right answer.

Reply:
Yeah, that would be a over-kill and bad ROI